src/UCommerce.SiteCore.Installer/UCommerce.SiteCore.Installer.csproj

```diff
@@ -73,7 +73,7 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="Infralution.Licensing, Version=4.7.1.0, Culture=neutral, PublicKeyToken=3e7e8e3744a5c13f">
-      <HintPath>..\packages\uCommerce.8.4.0.19331\lib\net45\Infralution.Licensing.dll</HintPath>
+      <HintPath>..\packages\uCommerce.8.4.1.19353\lib\net45\Infralution.Licensing.dll</HintPath>

```

Comment:
Is this part of the task? If yes, updating packages should have been a separate commit, since it creates lots of noise among which the actual fix is lost 😄

Reply:
It wasn't "part of the task", but David told me to add it to the PR as well, I don't feel qualified to make a decision on this at all, hah.

Reply:
Fair enough.